frontend/components/create-review/review-form.tsx

```diff
@@ -1,11 +1,236 @@
-import React from 'react'
+import React, {useState} from 'react'
+import Button from '../ui/button'
+import ButtonLight from '../ui/button-light'
+import RatingsRadio from './ratings-radio'
 
 //This components will hold the review form and it's data handling logic
 //Completed reviews should be sent to the backend with a success confirmation for the user (maybe need a Modal?)
 //Once completed, it should give an option to reset the form for another review or direct Client to Reviews page
 
+const zipRegex = /^[0-9]{5}(?:-[0-9]{4})?$/
+const postalRegex =
+	/^[ABCEGHJ-NPRSTVXY]\d[ABCEGHJ-NPRSTV-Z][ -]?\d[ABCEGHJ-NPRSTV-Z]\d$/i
+
+//TODO hook up with backend
+//TODO create error handling for regex tests
+
 function ReviewForm(): JSX.Element {
-	return <div></div>
+	const [landlord, setLandlord] = useState<string>('')
+	const [country, setCountry] = useState<string>('Canada')
```

Comment:
Lets pull from a full list of countries but limit it to Canada so we are prepared to transition to other spaces:
- https://github.com/umpirsky/country-list/blob/master/data/en/country.json

Also, use https://github.com/Cimpress-MCP/postal-codes-js to support validation against any country

Reply:
can do!

Reply:
how do you want me to pull in the list of countries into the form? Or just have it ready to go?

Reply:
Read from the json file but exclude everything but Canada and US use the country name as the display and the code as the value. That way we can remove the filter when needed